etna/metrics/base.py

```diff
@@ -128,7 +128,7 @@ class Metric(AbstractMetric, BaseMixin):
     def __init__(
         self,
         metric_fn: MetricFunction,
-        mode: str = MetricAggregationMode.per_segment,
+        mode: str = MetricAggregationMode.per_segment.value,
```

Comment:
We should consider setting here a plain string instead of an enum value

Reply:
You suggest using just "per-segment"?